vuepress/docs/next/tutorials/devops/installation/open-shift/openshift-install-by-operator-hub.md

```diff
@@ -262,7 +262,33 @@ Now we'll create the application itself.
 
 See the [Next Steps](#next-steps) below to continue your work with Entando.
 
+## Appendix A - Troubleshooting and Common Errors
+
+### SSL Certificate Error
+When installing Entando 6.3.2 into OpenShift 4.6, you may run into an SSL certificate error. This happens because of restricted registries for the docker image. 
+
+To address this issue, a property in the ConfigMap can be used to override the default docker registry.  Every time there is a docker image name without a registry, it will use this instead of the docker.io. 
```

Comment:
Minor, change "instead of the docker.io" to "instead of the default registry"  

In 6.3.2, the default is docker.io, in 7.0 it will be registry.hub.docker.com but the tip on how to change it is still useful.

Reply:
Replaced with "Every time there is a docker image without a registry, it will use this override property instead of the default." Registry was repetitive and this sounds better.